homeassistant/components/habitica/__init__.py

```diff
@@ -196,4 +292,5 @@ async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     """Unload a config entry."""
     if len(hass.config_entries.async_entries(DOMAIN)) == 1:
         hass.services.async_remove(DOMAIN, SERVICE_API_CALL)
+        hass.services.async_remove(DOMAIN, "cast_skill")
```

Comment:
Nope, we don't remove this one. We set it up in `async_setup` and add all the extra validation to be able to always provide the service call, instead of it being depending on the availability of a config entry, improving the way we validate the validity of automations